gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/CopyYourkitAgentTask.groovy

```diff
@@ -0,0 +1,30 @@
+package com.palantir.gradle.dist.service.tasks
+
+import com.palantir.gradle.dist.service.JavaServiceDistributionPlugin
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+
+import java.nio.file.Files
+import java.nio.file.Path
+
+class CopyYourkitAgentTask extends DefaultTask {
```

Comment:
This looks great so far! it would be great to have a unit test that verifies this file gets written (and is approximately the right size).

Hopefully should be able to use a similar structure to `ConfigTarTaskTest`. It uses spock rather than junit, so it all looks a bit alien, but you should be able to run your task from the unit test!

One key thing to assert is the incremental behaviour - we want to make sure this task understands it's up-to-date if the file already exists:
```
buildResult.task(':copyYourkitAgent').outcome == TaskOutcome.UP_TO_DATE
```